errors.js

```diff
@@ -0,0 +1,20 @@
+const ES6Error = require("es6-error");
+
+exports.onlyInstancesOf = function (errorType, handler) {
+  return (error) => {
+    if (error instanceof errorType) {
+      return handler(error);
+    // eslint-disable-next-line no-else-return
+    } else {
```

Comment:
@saintsebastian no need to an else block here (if you enter the if block you are going to return in any case. (that would probably "make eslint happy again" :-))